scripts/linux/pkg/deb/assets/test.sh

```diff
@@ -24,7 +24,7 @@ rvnpid=\$!
 
 trap "rc=\$?; pgrep \$rvnpid && kill -9 \$rvnpid 2>&1 >/dev/null; echo \$rc; exit \$rc" EXIT
 
-sleep 5 
+sleep 7 
```

Comment:
That looks suspicious and likely to fail, the `--rerty` isn't enough?

Reply:
`amd64` docker host via qemu is just slower than the other ones I presume.